e2e/README.md

```diff
@@ -24,78 +21,98 @@ existsnce, sysctl settings, etc.) is as expected.
 
-To run the Go implementation of the E2E test suite locally, simply use `e2e-local.sh`. This script will setup
-the `go test` command for you.
+To run the E2E test suite locally, use `e2e-local.sh`. This script sets up the `go test` command.
 
 Check [config.go](config/config.go) for the default configuration parameters. You can override these parameters by
-setting ENV variables or by temporary changing the code.
+setting ENV variables.
 
-`TAGS_TO_RUN=` may also optionally be set to specify a subset of the E2E scenarios to run based on the tags assigned to
-each scenario. If `TAGS_TO_RUN` is not specified, all scenarios will be ran. Multiple tags can be specified as a
-comma-separated list. In the case where multiple tags are specified, only scenarios which have all of the specified tags
-will be ran. Check scenario files for available tags. Tag names are case-insensitive.
+Create a `.env` file in the `e2e` directory to set environment variables and avoid manual setup each time you run tests. Refer to `.env.sample` for an example.
+
+### Running Specific Tests
+
+Use `TAGS_TO_RUN=` to specify scenarios based on tags. By default, all scenarios run. Multiple tags should be comma-separated and are case-insensitive. Check logs for test tags.
 
 Example:
 ```bash
-TAGS_TO_RUN="os=ubuntu2204,platform=arm64" ./e2e-local.sh
+TAGS_TO_RUN="os=ubuntu,arch=amd64,wasm=false,gpu=false,imagename=1804gen2containerd" ./e2e-local.sh
 ```
 
-Furthermore, `TAGS_TO_SKIP` may also optionally be set to specify the set of scenarios which will be excluded
-from the testing session as a commma-separated list. In the case where multiple tags are specified, only scenarios which
-have none of the specified tags will be ran (this logic is different to TAGS_TO_RUN).
+To exclude scenarios, use `TAGS_TO_SKIP=`. Scenarios with any specified tags will be skipped (this logic is different to TAGS_TO_RUN).
 
-If both `TAGS_TO_RUN` and `TAGS_TO_SKIP` are  specified, `TAGS_TO_SKIP` will take precedence.
+To run a specific test, use the test name:
+```bash
+TAGS_TO_RUN="name=Test_azurelinuxv2" ./e2e-local.sh
+# or
+go test -run Test_azurelinuxv2 -v -timeout 20m
+```
 
-`KEEP_VMSS` can also be optionally specified to have the test suite retain the bootstrapped VM(s) for further debugging.
-When this option is specified, the private SSH key used to connect to each VM will be included within each scenario's
-log bundle respectively.
 
-**Note that when using `e2e-local.sh`, a timeout value of 90 minutes is applied to the `go test` command.**
+### Debugging
 
-You may also run the test command with custom arguments yourself (assuming you've properly setup the required
-environment variables) from within the `e2e/` directory like so:
+Set `KEEP_VMSS` to retain bootstrapped VMs for debugging. This will include the private SSH key in each scenario's log bundle. When using the flag, please ensure to run only test you need to debug, as the VMs will not be
```

Comment:
```suggestion
Set `KEEP_VMSS=true` to retain bootstrapped VMs for debugging. Setting this will also have the VM's private SSH key included in each scenario's log bundle. When using this flag, please ensure to run only test you need to debug, as the VMs will not be
```